src/main/java/games/strategy/engine/lobby/server/db/DbUserController.java

```diff
@@ -67,6 +80,12 @@ public void updateUser(final DBUser user, final HashedPassword password) {
     secondaryDao.updateUser(user, password);
   }
 
+  @Override
+  public void updateUser(final DBUser user, final String password) {
```

Comment:
While this is possible for createUser and updateUser it is not possible for login, because the hash needs to be recalculated using the "unhashed" password + stored salt.